test/truefi/TrueFiPool.test.ts

```diff
@@ -77,7 +76,7 @@ describe('TrueFiPool', () => {
 
   it('cannot exit and join on same transaction', async () => {
     const arbitrage = await new PoolArbitrageTestFactory(owner).deploy()
-    await token.transfer(arbitrage.address, parseEther('1'))
+    await token.transfer(arbitrage.address, parseEth('1'))
```

Comment:
```suggestion
    await token.transfer(arbitrage.address, parseEth(1))
```
just to keep up with new convention